shared/src/commonTest/kotlin/co/nimblehq/blisskmmic/data/datasource/LocalDataSourceTest.kt

```diff
@@ -0,0 +1,70 @@
+package co.nimblehq.blisskmmic.data.datasource
+
+import co.nimblehq.blisskmmic.data.database.core.DataStore
+import co.nimblehq.blisskmmic.data.database.datasource.LocalDataSourceImpl
+import co.nimblehq.blisskmmic.data.database.model.TokenDatabaseModel
+import io.kotest.matchers.shouldBe
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.test.runTest
+import org.kodein.mock.Fake
+import org.kodein.mock.Mock
+import org.kodein.mock.tests.TestsWithMocks
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+
+@ExperimentalCoroutinesApi
+class LocalDataSourceTest: TestsWithMocks() {
+
+    @Mock
+    lateinit var dataStore: DataStore
+    @Fake
+    lateinit var tokenDB: TokenDatabaseModel
+
+    private val dataSource by withMocks { LocalDataSourceImpl(dataStore) }
+
+    override fun setUpMocks() = injectMocks(mocker)
+
+    @BeforeTest
+    fun setUp() {
+        mocker.reset()
+    }
+
+    @Test
+    fun `When calling save token, it calls dataStore to save with correct key and value`() = runTest {
```

Comment:
Small naming suggestion to avoid repeated wording usage 🙏
```suggestion
    fun `When saving token, it saves the correct key and value to dataStore`() = runTest {
```